### DISCUSSION TITLE
mandate CognitiveAtlas tagging for Projects

### DISCUSSION THREAD

**Discussion by adelavega** [neurostuff/neurostore#1023] [Ideas]
We need to mandate that users tags Projects or at least a subset of studies in a meta-analysis with a CognitiveAtlas tag.

I think for now we can do the lazy thing like NeuroVault which shows the existing tags and "None" if nothing fits

**Comment by adelavega** [comment]
Thinking about this more, my concern is that Cognitive Atlas is pretty incomplete. If we incorporate this, we need to make it easy to modify CogAtlas which is an issue rn. 

Now that we're incorporating AI features, I think we need to re-think this to instead have users accept or reject AI tags on studies/meta-analysis. We can also determine what a meta-analysis is about based on which studies are included.

**Comment by jdkent** [comment]
I'll move this to discussion, I agree, there is more effort needed to make cognitive atlas editable if we want to adopt it here.
